Cargo.lock

```diff
@@ -1264,31 +1274,30 @@ checksum = "487585f4d0c6655fe74905e2504d8ad6908e4db67f744eb140876906c2f3175d"
 dependencies = [
  "proc-macro2",
  "quote",
- "syn 2.0.22",
+ "syn 2.0.27",
 ]
 
 [[package]]
 name = "ed25519"
-version = "1.5.3"
+version = "2.2.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "91cff35c70bba8a626e3185d8cd48cc11b5437e1a5bcd15b9b5fa3c64b6dfee7"
+checksum = "5fb04eee5d9d907f29e80ee6b0e78f7e2c82342c63e3580d8c4f69d9d5aad963"
 dependencies = [
+ "pkcs8",
```

Comment:
Is this a sensible extra dependency, or maybe something upstream should make optional?

Reply:
It is already optional, and we don't enable it :thinking: 

https://github.com/dalek-cryptography/curve25519-dalek/blob/e44d4b5903106dde0e5b28a2580061de7dfe8a9f/ed25519-dalek/Cargo.toml#L71

Reply:
Hm, I can't find out why it would be enabled. Maybe it actually isn't, but Cargo still puts it in the lockfile for some reason.